cdap-hbase-compat-0.96/src/main/java/co/cask/cdap/data2/util/hbase/HBase96TableUtil.java

```diff
@@ -255,6 +256,28 @@ public BloomType getBloomFilter(HColumnDescriptor columnDescriptor) {
     return datasetStat;
   }
 
+  @Override
+  public String getSysConfigTablePrefix(String tableName) {
+    TableName hTableName = TableName.valueOf(tableName);
+    String rootPrefix;
+    if (Constants.DEFAULT_NAMESPACE.equals(hTableName.getNamespaceAsString())) {
+      // Input: cdap.table (in default namespace)
+      // Output: cdap_system:
```

Comment:
having trouble understanding how this will output cdap_system (after we change prefixes to not be hardcoded).  Are we just leaving this logic half-implemented?